astropy/units/format/generic.py

```diff
@@ -529,18 +534,11 @@ def _parse_unit(cls, s: str, detailed_exception: bool = True) -> UnitBase:
     _regex_superscript: ClassVar[Pattern[str]] = re.compile(
         f"[{_superscripts}]?[{_superscripts[2:]}]+"
     )
-    _regex_deg: ClassVar[Pattern[str]] = re.compile("°([CF])?")
 
     @classmethod
     def _convert_superscript(cls, m: Match[str]) -> str:
         return f"({m.group().translate(cls._superscript_translations)})"
 
-    @classmethod
-    def _convert_deg(cls, m: Match[str]) -> str:
-        if len(m.string) == 1:
```

Comment:
This was actually wrong, it should have been `m.group()`. Hence, for "°C/s",  "°C" would be substituted with "deg_C/s" giving a full string of "deg_C/s/s".